galaxy/values.yaml

```diff
@@ -443,7 +480,8 @@ configs:
       interactivetools_base_path: "{{$host := index .Values.ingress.hosts 0}}{{$path := index $host.paths 0}}{{$path.path}}"
       id_secret:
       mulled_resolution_cache_lock_dir: "/galaxy/server/local/mulled_cache_lock"
-      database_connection: postgresql://unused:because@overridden_by_envvar
+      database_connection: |-
+        {{ include "galaxy-postgresql.connection-string" .}}
```

Comment:
This was set to a constant on purpose as I recall, so that the credentials do not get embedded in a plaintext file.

Reply:
Per the discussion on Slack I will *resolve* this and leave as is until the Galaxy maintenance script properly support `GALAXY_CONFIG_OVERRIDE_*` variables.